ddtrace/contrib/flask/middleware.py

```diff
@@ -188,11 +188,15 @@ def _patch_render(tracer):
     # fall back to patching  global method
     _render = flask.templating._render
 
+    if hasattr(_render, "__dd_orig"):
```

Comment:
why don't we just return here? if it is already patched, why re-patch?

Reply:
The info was in the commit 😄.

> We can't depend on a flag since the patching could introduce a new tracer instance.

The patching installs a given tracer instance so if the middleware is installed again it won't use the new tracer instance. So we need to re-instrument.

Reply:
maybe it needs a comment ;)

Reply:
done